stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -713,6 +714,26 @@ internal class ChatDomainImpl internal constructor(
         return messages
     }
 
+    @VisibleForTesting
+    internal suspend fun retryReactions(): List<Reaction> {
+        return repos.selectReactionSyncNeeded().onEach { reaction ->
+            reaction.user = null
```

Comment:
Why do we have to remove a user from the reaction before updating it on the backend?

Reply:
I actually don't know it was copy-pasted from `ReactionRepository`. If you're sure that it's not right, we can change it

Reply:
I'm not sure about that. Let's leave it for now but maybe it would be worth it to investigate that